src/prepare.rs

```diff
@@ -68,6 +70,14 @@ impl<'a> Prepare<'a> {
         Ok(())
     }
 
+    fn remove_cargo_config(&self) -> Result<(), Error> {
+        let path = self.source_dir.join(".cargo").join("config");
+        if path.exists() {
+            remove_file(path)?;
```

Comment:
Can you add an `info!` here that says the `.cargo/config` file was deleted? I'd prefer for all the changes done during the prepare to be logged, so it's easy to see what changed between the local build and the docs.rs build when debugging build differences.